src/configuration.cpp

```diff
@@ -0,0 +1,702 @@
+                   .set_env_var_name()
+                   .description("Permit use of the --overide-channels command-line flag"));
+
+        // Network
+        insert(Configurable("local_repodata_ttl", &ctx.local_repodata_ttl)
+                   .group("Network")
+                   .description("Repodata time-to-live")
+                   .long_description(unindent(R"(
+                        For a value of 0, always fetch remote repodata (HTTP 304
+                        responses respected).
+                        For a value of 1, respect the HTTP Cache-Control max-age header.
+                        Any other positive integer values is the number of seconds to
+                        locally cache repodata before checking the remote server for
+                        an update.)")));
+
+        insert(Configurable("ssl_verify", &ctx.ssl_verify)
+                   .group("Network")
+                   .set_env_var_name()
+                   .description("Verify SSL certificates for HTTPS requests")
+                   .long_description(unindent(R"(
+                        'ssl_verify' can be either an empty string (regular SSL verification),
+                        the string "<false>" to indicate no SSL verification, or a path to
+                        a directory with cert files, or a cert file..)"))
+                   .set_post_build_hook(detail::ssl_verify_hook));
+
+        insert(Configurable("cacert_path", std::string(""))
+                   .group("Network")
+                   .set_env_var_name()
+                   .description("Path (file or directory) SSL certificate(s)")
+                   .long_description(unindent(R"(
+                        Path (file or directory) SSL certificate(s) to use whe
+                        'ssl_verify' in turned on but not set with path to certs.
+                        WARNING: overrides 'ssl_verify' if provided and 'ssl_verify'
+                        also contains a path to SSL certificates.)")));
+
+        insert(Configurable("ssl_no_revoke", &ctx.ssl_no_revoke)
+                   .group("Network")
+                   .set_env_var_name()
+                   .description("SSL certificate revocation checks")
+                   .long_description(unindent(R"(
+                        This option tells curl to disable certificate revocation checks.
+                        It's only working for 'Schannel' back-end.
+                        WARNING: this option loosens the SSL security, and by using this
+                        flag you ask for exactly that.)")));
+        // Solver
+        insert(Configurable("channel_priority", &ctx.channel_priority)
+                   .group("Solver")
+                   .set_env_var_name()
+                   .description("Define the channel priority ('strict', 'flexible' or 'disabled')")
+                   .long_description(unindent(R"(
+                    Accepts values of 'strict', 'flexible', and 'disabled'. The default
```

Comment:
actually, we don't really implement anything for "flexible", only strict and disabled.

Reply:
You're right, I modified the description.
I also remove `strict_channel_priority` from the context since what's matter is `channel_priority`.
If we don't want to implement later a `flexible` option, we can also keep the former and remove `channel_priority` ?